docs/api-docs/tax/tax-settings.mdx

```diff
@@ -2,6 +2,12 @@
 
 Tax settings allow you to choose how a store handles tax calculation and price display. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on orders and invoices. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
 
+## Estimate taxes and display prices based on tax zone
+
+The Tax Settings API supports cross-border transactions. You can localize storefront prices for shoppers from different regions. For example, you can display tax-inclusive or exclusive prices depending on the tax zone. This benefits single-storefront stores that sell to shoppers from different regions. This functionality applies to Stencil and GraphQL-powered storefronts.
+
+To support cross-border transactions, the store identifies a tax zone for the shopper. The store then applies the tax zone and its settings to estimate taxes and display prices. For logged-in customers, the store uses the customer address to identify the tax zone. For guest shoppers, the store can use the shopper's geolocation to identify the zone if you enable this setting. Otherwise, the store uses the provided guest shopper tax zone ID.
```

Comment:
Only comments that you can take or not here is the use of "the store can, the store then, the store uses." Maybe we can say the store is set up to, or something similar to avoid anthropomorphism?

Reply:
I think the active verbs make it easier to understand, so I'll keep it for now